Added characters for the Phantom Jungle adventure

SageThalcos opened the pull request:
Added characters for the 1shotadventures Phantom Jungle adventure

ethorad reviewed (changes requested):
I have looked through the character compared to the GURPS Cliffhangers link on https://1shotadventures.com/adventure-index/
However this doesn't have all of the characters in this addition.  Where do the others come from?

I have noted against each file where GCS differs from the sheet.  Some of these should be fixed, some could be changed, and some look like errors in the source sheet.

Where there are errors in the source sheet, and GCS ends up a bit different, you could consider adding a note in GCS commenting on the change.  That way users who spot the difference will know why.  Up to you.

SageThalcos commented:
Updated the characters with the above notes. (Note that I'm the author of the adventure, so I also fixed issues in the .pdf version of the characters in some cases, as the GCS files were the "authoritative" vs. the pdfs.) Also, this pull request includes the NPCs in the adventure (found in the VTT files on 1shotadventures.com), which folks have asked for for inclusion into GCS, to more easily tweak them for their games.

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Chio Cervantes.gcs:
Fixed. Note that the GCS file was the correct version, so I updated the .pdf on 1shotadventures.com

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Sônia Carrilho.gcs:
Fixed, and also updated the .pdf to fix the Dodge error.

ethorad replied:
Thanks, all looks good

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Willy Van der Woodson.gcs:
Fixed and unified both the gcs and pdf versions to be at 150 points

ethorad replied:
Sheet has Overconfidence [-10] but GCS has Overconfidence [-5].  I think this needs a correction on the sheet, as the points add to 150 on GCS.
Could add the CR on the sheet to help users?

SageThalcos replied:
Yes, this is a mistake on the sheet, should be -5. Fixed in the pdf. (A lot of newer GURPS players have told me the CR rolls confuse them on the sheets, so I only put them in if they are different than normal now)

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Marisa Santarelli.gcs:
Fixed. Some price discrepancies were due the pricing in the original GURPS Cliffhangers, but I unified most everything to GCS/Basic Set/High Tech

ethorad replied:
If you can change the pdfs, Perception point cost should be 0 (as it's equal to IQ)

On GCS there's 1 point in Climbing and 1 in Performance, however the sheet has 2 in Performance (and no Climbing)

ethorad replied:
If you can change the pdf, Perception should be 0 points as it's equal to IQ (11).  Otherwise happy that GCS matches the sheet.

SageThalcos replied:
Fixed in the pdf.

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Jonny Talon.gcs:
Addressed issues in the new version.

ethorad replied:
Happy with this, thanks

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Duke Duckworth.gcs:
Addressed issues in both the GCS and the adventure pdf.

ethorad replied:
Happy they match up, thanks

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Patience Paquet.gcs:
Addressed issues in both the GCS and the adventure pdf so she adds to 150 points.

ethorad replied:
Happy with the changes, all matches up

ethorad commented:
I see the various GCS files in the VTT assets.  I assume that these are the same files as are in the library so I'll just have a look over them for structure / consistency rather than checking they match.  That OK?

I've been through your changes to the 7 PCs, will have a look at the NPCs next chance I get.

ethorad commented:
One thought was since these are yours, you could consider adding the portrait pictures onto the GCS files?  Didn't mention it earlier as I hadn't realised these were yours so assumed you didn't have the copyright to the pictures.

ethorad reviewed (commented):
Just a couple of minor changes, thanks

SageThalcos commented on Library/1shotadventures/Characters/The Phantom Jungle/Papa Mook Zevallo.gcs:
He doesn't really use one in the adventure, but for completeness I gave him a sidearm.

ethorad replied:
Ah, fair point.  At least this now gives him a chance against pesky PCs :)

SageThalcos reviewed (commented):
Addressed all outstanding issues for Phantom Jungle characters

ethorad commented on Library/1shotadventures/Characters/The Phantom Jungle/Cornelius Bee.gcs:
At the moment, it's showing up as damage of "(100)(+1 per die) cr".  I think using 100 as the armor divisor is fine, I can't see a way to say "ignores armor".  The +1 per die is a little odd though.  Instead do the boxes as:
Damage: None
First box: +1
Next box (just before the "per die" label): +0

That should then show up as damage of "4 (100) cr" on the sheet

ethorad replied:
Looks good, thanks

ethorad commented on Library/1shotadventures/Characters/The Phantom Jungle/Kirby Bell Bee.gcs:
Looks good

ethorad reviewed (changes requested):
Just a tweak to the TK damage i think

SageThalcos commented:
> Just a tweak to the TK damage i think

Fixed!

SageThalcos reviewed (commented):
Fixed TK problem and added portraits to all NPCs.

ethorad reviewed (approved):
Thanks, merging changes into the master library
